src/main/java/io/github/intellij/dlanguage/DlangSdkType.java

```diff
@@ -83,17 +93,17 @@ public class DlangSdkType extends SdkType {
             };
         } else if (SystemInfo.isUnix) {
             // The official .rpm and .deb installers are the priority
-            DEFAULT_DMD_PATHS = new File[] {
-                new File("/usr/bin/dmd"), // Fedora (official .rpm), note that Arch also uses this path
-                new File("/usr/local/bin/dmd"), // Ubuntu
-                new File("/snap/bin/dmd") // snapcraft.io (symlink to /snap/dmd/current/bin/dmd)
-            };
-            // the path to D documentation should contain "index.html"
-            DEFAULT_DOCUMENTATION_PATHS = new File[] {
-                new File("/usr/share/dmd/html/d"), // Fedora (official .rpm)
-                new File("/usr/local/share/dmd/html/d"), // Ubuntu
-                new File("/usr/share/d/html/d") // Arch Linux (dmd-docs package is not consistent with dlang & dlang-dmd package paths)
-            };
+//            DEFAULT_DMD_PATHS = new File[] {
+//                new File("/usr/bin/dmd"), // Fedora (official .rpm), note that Arch also uses this path
+//                new File("/usr/local/bin/dmd"), // Ubuntu
+//                new File("/snap/bin/dmd") // snapcraft.io (symlink to /snap/dmd/current/bin/dmd)
+//            };
+//            // the path to D documentation should contain "index.html"
+//            DEFAULT_DOCUMENTATION_PATHS = new File[] {
+//                new File("/usr/share/dmd/html/d"), // Fedora (official .rpm)
+//                new File("/usr/local/share/dmd/html/d"), // Ubuntu
+//                new File("/usr/share/d/html/d") // Arch Linux (dmd-docs package is not consistent with dlang & dlang-dmd package paths)
+//            };
             // the path to phobos should contain "etc.c.*" and "std.*"
             DEFAULT_PHOBOS_PATHS = new File[] {
```

Comment:
They need to be updated per SDK type to point to the phobos and druntime provided by the compiler.

Reply:
that's in progress at minute. Was hoping to get done at weekend but the changes for release took longer than expected. It'll be a while before this pr is ready.